test/e2e/framework/framework.go

```diff
@@ -52,7 +52,7 @@ import (
 
 const (
 	maxKubectlExecRetries           = 5
-	DefaultNamespaceDeletionTimeout = 5 * time.Minute
+	DefaultNamespaceDeletionTimeout = 10 * time.Minute
```

Comment:
add a comment referring to this PR and the associated issue and indicate it should be returned to 5 minutes once the issue is resolved